rasgotransforms/rasgotransforms/transforms.py

```diff
@@ -0,0 +1,113 @@
+import os
+from typing import Callable, Dict, Optional, Union
+from pathlib import Path
+import inspect
+import yaml
+
+from .jinja import render
+from .main import DataWarehouse
+
+
+class Transforms(object):
+    """
+    Dynamically adds class methods which render each transform in the package
+    """
+
+    def __init__(self):
+        transforms_dir = Path(os.path.dirname(__file__), 'transforms')
```

Comment:
```suggestion
        transforms_dir = Path(__file__).parent / 'transforms'
```

Can remove the `os` module and just switch to using `pathlib` throughout the file